src/app/components/Columns/BookmarkColumn.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import ShowBookmarkColumnHead from '../../containers/Columns/ShowBookmarkColumnHead'
+import DndBookmarkColumnCover from '../../containers/Columns/DndBookmarkColumnCover'
+
+// Visualisation of the column for bookmark cards
+const BookmarkColumn = ({ className, id, name, cards }) => (
```

Comment:
Functional components doesnt give you the opportunity to implement a shouldComponentUpdate. So everytime the parent is updating, this bookmark component is updating too.

As long React doesnt consider `shouldComponentUpdate` for functional components, you should use functional components only for displaying ui-data. But its more usecase specific